enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -23,13 +23,33 @@ configure and connect the Superhub and Feeder hubs. For Feeder hubs with an
 earlier version than 3.14.0 some manual steps must be taken. Links to these
 are provided at each stage of installation and setup that follows.
 
+* [Requirements][Federated Reporting#Requirements]
 * [Installation][Federated Reporting#Installation]
 * [Setup][Federated Reporting#Setup]
 * [Operation][Federated Reporting#Operation]
 * [API Setup][Federated Reporting#API Setup]
 * [Disable Feeder][Federated Reporting#Disable Feeder]
 * [Uninstall][Federated Reporting#Uninstall]
 
+## Requirements ##
+
+If your hub will have SELinux enabled you must install the `semanage` command.
+This allows Federated Reporting policy to manage the trust between the superhub and
+feeder hubs.
+
+Add the `cfengine_mp_fr_dependencies_auto_install` to your augments file to allow
```

Comment:
this might be in line with how we doc other things, but to me, it is not clear how to achieve what you are explaining/requesting the user to do.

Reply:
Ah. I have had this conversation with @nickanderson before maybe. "add to augments" is really the phrase and it means generally editing `/var/cfengine/masterfiles/def.json`.

Reply:
https://docs.cfengine.com/docs/3.12/reference-language-concepts-augments.html